### ISSUE TITLE
ERROR: jsonParse is not defined on AWS SNS Trigger

### ISSUE THREAD

**Issue by luizeof** [n8n-io/n8n#4461]
**Describe the bug**

Error receiving data in AWS SNS Trigger:

```javascript
ReferenceError: jsonParse is not defined
    at Object.webhook (/usr/local/lib/node_modules/n8n/node_modules/n8n-nodes-base/dist/nodes/Aws/AwsSnsTrigger.node.js:132:22)
    at Workflow.runWebhook (/usr/local/lib/node_modules/n8n/node_modules/n8n-workflow/dist/src/Workflow.js:591:33)
    at Object.executeWebhook (/usr/local/lib/node_modules/n8n/dist/src/WebhookHelpers.js:109:48)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

![727ae953-1a57-4eeb-9c1c-7e874f3039c2](https://user-images.githubusercontent.com/257004/198494525-4383278c-624f-4148-9228-517ed1776286.png)

**To Reproduce**

1. Upgrade N8N to 0.200
2. Receive data on AWS SNS Trigger

**Expected behavior**

**Environment (please complete the following information):**

- OS Docker
- n8n Version 0.200.0
- Node.js Version [e.g. 14.16.0]
- Database system [e.g. SQLite; n8n uses SQLite as default otherwise changed]
- Operation mode [e.g. own; operation modes are `own`, `main` and `queue`. Default is `own`]

**Additional context**

**Comment by luizeof** [comment]
@Joffcom looks like we have a little bug here.

**Comment by adam91holt** [comment]
I'm also getting this too after updating to 0.200.0

**Comment by Joffcom** [comment]
Hey @luizeof,

I will take a look at this first thing, for now the best option would be to drop down to 0.199.0.

**Comment by amirhmoradi** [comment]
I am also getting this error, will downgrade to 199 waiting for a fix.

**Comment by janober** [comment]
Fix got released with `n8n@0.200.1`
